_sections/30-bindings.md

```diff
@@ -172,7 +172,7 @@ For convenience, the functions are categorized based on their main usage. Some f
 <a id="fn:format-date" href="#fn:format-date">`format-date(date value, string format)`</a> | string | Returns the provided date value formatted as defined by the format argument using the following identifiers:<br/>`%Y`: 4-digit year<br/>`%y`: 2-digit year<br/>`%m` 0-padded month<br/>`%n` numeric month<br/>`%b` short text month (Jan, Feb, etc)\*<br/>`%d` 0-padded day of month<br/>`%e` day of month<br/>`%a` short text day (Sun, Mon, etc).\* <br/>\* If form locale can be determined that locale will be used. If form locale cannot be determined the locale of the client will be used (e.g. the browser or app).
 <a id="fn:format-date-time" href="#fn:format-date-time">`format-date-time(dateTime value, string format)`</a> | string | Returns the provided dateTime value formatted as defined by the format argument using the same identifiers as [`format-date`](#fn:format-date) plus the following:<br/>`%H` 0-padded hour (24-hr time)<br/>`%h` hour (24-hr time)<br/>`%M` 0-padded minute<br/>`%S` 0-padded second<br/>`%3` 0-padded millisecond ticks.\* <br/>\* If form locale can be determined that locale will be used. If form locale cannot be determined the locale of the client will be used (e.g. the browser or app).
 <a id="fn:date" href="#fn:date">`date(* value)`</a> | string | Converts to a string in the ....date format.
-<a id="fn:decimal-date-time" href="#fn:decimal-date-time">`decimal-date-time(dateTime value)`</a> | number | Converts dateTime value to the number of days since January 1, 1970 UTC. This is the format used by Excel.
+<a id="fn:decimal-date-time" href="#fn:decimal-date-time">`decimal-date-time(dateTime value)`</a> | number | Converts dateTime value to the number of days since January 1, 1970 UTC.
```

Comment:
:+1: might be good to emphasise **days**, as many might misread this as being a unix timestamp.

Reply:
https://github.com/opendatakit/xforms-spec/pull/154 Is the discussion on the topic with several other wordings that we tried. Perhaps we can file a separate issue to discuss further clarification?